packages/astro/src/integrations/index.ts

```diff
@@ -34,21 +35,22 @@ async function withTakingALongTimeMsg<T>({
 }
 
 export async function runHookConfigSetup({
-	config: _config,
+	settings,
 	command,
 	logging,
 }: {
-	config: AstroConfig;
+	settings: AstroSettings;
```

Comment:
This is the only area where I'm concerned about potential regressions. Is our test coverage for these hooks sufficient?

Reply:
This doesn't change the signature of hooks, this is just the function that calls the hooks.